apache-rat-core/src/main/java/org/apache/rat/document/DocumentNameMatcher.java

```diff
@@ -400,9 +399,8 @@ public String toString() {
     interface CollectionPredicate extends Predicate<DocumentName> {
         Iterable<DocumentNameMatcher> getMatchers();
     }
-
     /**
-     * A marker class to indicate this predicate contains a collection of matchers.
+     * CollectionPredicate implementation.
      */
     abstract static class CollectionPredicateImpl implements CollectionPredicate {
```

Comment:
@Claudenw would DefaultCollectionPredicate be a better name for the implementation? or RatCollectionPredicate instead?